workspace/pipeline/pln_trigger_function_app.json

```diff
@@ -97,12 +97,53 @@
 					"setSystemVariable": true
 				}
 			},
+			{
+				"name": "Set Secret Name",
+				"type": "SetVariable",
+				"dependsOn": [],
+				"policy": {
+					"secureOutput": true,
+					"secureInput": false
+				},
+				"userProperties": [],
+				"typeProperties": {
+					"variableName": "secret_name",
+					"value": {
+						"value": "@concat('function-url-', replace(pipeline().parameters.function_name, '-', ''))",
+						"type": "Expression"
+					}
+				}
+			},
+			{
+				"name": "Set Function Secret",
```

Comment:
I don't think anything in this file needed to be changed? This pipeline was ok, it just needed parameters to be passed to it?